bfly/CoreLayer/AccessLayer/QueryLayer/UtilityLayer/Keywords.py

```diff
@@ -153,7 +153,7 @@ def __init__(self):
         # ALL THE DATABASE RUNTIME TERMS
         self.DB = NamelessStruct(
             TABLE = NamelessStruct(
-                LIST = _table_list,
+                LIST = _table_list[:],
```

Comment:
We do not want to write this for each list in `UtilityLayer/Keywords.py`, since it will be very easy to forget. The dereferencing with `[:]` should instead happen in `UtilityLayer/Structures.py`, where it will only need to be written once in the class definition of `NamelessStruct`.